homeassistant/components/linkplay/entity.py

```diff
@@ -46,7 +46,9 @@ def __init__(self, bridge: LinkPlayBridge) -> None:
 
         self._attr_device_info = dr.DeviceInfo(
             configuration_url=bridge.endpoint,
-            connections={(dr.CONNECTION_NETWORK_MAC, bridge.device.properties["MAC"])},
+            connections={
+                (dr.CONNECTION_NETWORK_MAC, bridge.device.properties.get("MAC", None))
+            },
```

Comment:
Instead we should not set connections if not MAC is present as a value needs to be present.